app/views/admins/_user_entry.haml

```diff
@@ -1,70 +1,81 @@
-
 %li.user.media
-  %div.pull-left
+  .media-left
     - if user.person
-      %span.media-object
+      .media-object
         = person_image_tag(user.person, size: :thumb_small)
-
-  %div.media-body.row
-    %div.pull-right
-      %span.label.label-default
-        = t('.id')
-        = user.id
-      %span.label.label-info
-        = t('.guid')
-        = user.person.guid if user.person
-
-    %h4.media-heading
-      = user.person.name if user.person
-
-    %div.pull-right
-      .unstyled.text-right.actions
-        = link_to t('admins.user_search.view_profile'), person_path(user.person), class: 'btn btn-default btn-block btn-xs'
-        = link_to t('admins.user_search.add_invites'), add_invites_path(user.invitation_code), class: 'btn btn-info btn-block btn-xs'
-        - unless user.person.closed_account
-          = link_to t('admins.user_search.close_account'), admin_close_account_path(user), method: :post, data: { confirm: t('admins.user_search.are_you_sure') }, class: 'btn btn-danger btn-block btn-xs'
-
-        - unless user.closed_account?
-          - unless user.access_locked?
-            = link_to t('admins.user_search.lock_account'), admin_lock_account_path(user), method: :post, data: { confirm: t('admins.user_search.are_you_sure_lock_account') }, class: 'btn btn-danger btn-block btn-xs'
-          - else
-            = link_to t('admins.user_search.unlock_account'), admin_unlock_account_path(user), method: :post, data: { confirm: t('admins.user_search.are_you_sure_unlock_account') }, class: 'btn btn-danger btn-block btn-xs'
-
-    %div.row
-      %div.col-md-5
+  .media-body
+    .row
+      .col-sm-7
+        %h4.media-heading
+          = user.try(:person).try(:name)
+      .col-sm-5
+        .pull-right
+          %span.label.label-default
+            = t(".id")
+            = user.id
+          %span.label.label-info
+            = t(".guid")
+            = user.try(:person).try(:guid)
+    .row
+      .col-sm-8
         %dl.dl-horizontal
-          %dt= t('username')
+          %dt= t("username")
           %dd= user.username
-          %dt= t('.email')
+          %dt= t(".email")
           %dd= user.email
-          %dt= t('.diaspora_handle')
+          %dt= t(".diaspora_handle")
```

Comment:
Yes, the key shouldn't be changed (otherwise we need to re-translate it), but the translation should be "diaspora\* ID".